src/js/components/TaskDetail.js

```diff
@@ -27,14 +32,14 @@ const METHODS_TO_BIND = [
   'onTaskDirectoryStoreSuccess'
 ];
 
-class TaskSidePanelContents extends SidePanelContents {
+class TaskDetail extends mixin(InternalStorageMixin, TabsMixin, StoreMixin) {
   constructor() {
     super(...arguments);
 
     this.tabs_tabs = Object.assign({}, TABS);
 
     this.state = {
-      currentTab: Object.keys(this.tabs_tabs).shift(),
+      currentTab: 'details',
```

Comment:
When you iterate over an Objects keys they will be returned in insertion order which is the same as the order declared when creating the Object. e.g. `{first:.., second:..}`. So we should be OK because we don't mutate the `tabs_tabs` Object in a way that would disrupt the order :)